### DISCUSSION TITLE
Disappearing body parts.

### DISCUSSION THREAD

**Discussion by MysticPhoenix22** [Monika-After-Story/MonikaModDev#10033] [QA - tech support + general] [answered]
At random times, Monika's body parts would just disappear for no reason. Mostly her face but sometimes her arms just go poof too. 
![Screenshot (9)](https://user-images.githubusercontent.com/111782262/218793018-b01f8ede-a40a-4871-be10-3dd8b8ee6954.png)

**Comment by Booplicate** [comment, marked as answer]
This is potentially an issue with your drive. Alternatively, I've seen this happening when people minimize the game.

**Comment by Nomireligion** [comment]
that's fucking terrifying
